RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -1755,65 +1382,32 @@ End Sub</value>
   <data name="RegexAssistant_IgnoreCaseFlag" xml:space="preserve">
     <value>Ignorer la casse</value>
   </data>
-  <data name="HotkeyDescription_SourceControl" xml:space="preserve">
-    <value>Contrôle des sources</value>
-  </data>
   <data name="TestExplorerMenu_AddExpectedErrorTestMethod" xml:space="preserve">
     <value>Méthode de test (&amp;erreur attendue)</value>
   </data>
-  <data name="SortStyle_ByCodeOrder" xml:space="preserve">
-    <value>Tel que dans le module</value>
-  </data>
   <data name="RubberduckReloadFailure_Message" xml:space="preserve">
-    <value>Les fenêtres ancrables n'ont probablement pas été correctement détruites, et/ou n'ont pas pu être recréées. l'environnement peut ne pas être dans un état stable. Rubberduck démerrera normalement à la prochaine initialisation du VBE.</value>
+    <value>Les fenêtres ancrables n'ont probablement pas été correctement détruites et/ou n'ont pas pu être recréées. L'environnement peut ne pas être dans un état stable. Rubberduck démarrera normalement à la prochaine initialisation du VBE.</value>
```

Comment:
I wonder if we'll ever see this msgbox again, now that reloading "works" :smile: